tests/integration/profile.test.ts

```diff
@@ -36,32 +41,55 @@ describe("profile", () => {
     return profile
   }
 
-  it("Sets the display name and role for new users", async () => {
+  async function testDBConnection() {
+    const newUser = fakeUser()
+    await setNewProfile(newUser)
+    let profile = await getProfile(newUser)
+    return profile
+  }
+
+  it("tests db conn", async () => {
+    const profile = await testDBConnection()
+    expect(profile).toBeDefined()
+  })
+
+  it("Sets the fullName for new users", async () => {
     const expected = fakeUser()
-    await expect(getProfile(expected)).resolves.toBeUndefined()
+    expect(await getProfile(expected)).toBeUndefined()
     const profile = await expectProfile(expected)
     expect(profile.fullName).toEqual(expected.fullName)
-    expect(profile.role).toEqual("user")
+    expect(profile.role).not.toBeDefined()
   })
 
-  it("Is not publicly readable by default", async () => {
+  it("Is publicly readable by default", async () => {
+    // this req was updated in June 2023
```

Comment:
hm maybe not necessary? Is there a reason to mark the date of this request? Maybe if it is we should link to the original issue?